cmd/osbuild-worker/main.go

```diff
@@ -84,6 +90,76 @@ func WatchJob(ctx context.Context, job worker.Job) {
 	}
 }
 
+// protect an AWS instance from scaling and/or terminating.
+func setProtection(protected bool) {
+	// create a new session
+	awsSession, err := session.NewSession()
+	if err != nil {
+		logrus.Debug("impossible to get an AWS session")
```

Comment:
Nitpick, but instead of "impossible" how about "Error (getting...)". Just to align more with the rest of the codebase.

Reply:
I kept the Debug level so that it does not pollute the log output.

Reply:
fixed now.